src/HumusAmqpModule/Consumer.php

```diff
@@ -263,42 +263,40 @@ public function consume($msgAmount = 0)
     {
         $this->target = $msgAmount;
 
-        foreach ($this->queues as $index => $queue) {
-            if (!$this->timestampLastAck) {
-                $this->timestampLastAck = microtime(1);
-            }
-
-            $message = $queue->get();
+        /** @var \AMQPQueue $queue */
+        foreach($this->queues as $queue) {
+            $queue->consume(function ($message, $queue) {
```

Comment:
How you implemented this in 1.0 version?
Actually we need only one queue to be cinsumed. Common usage is 1 process = 1 consumer to 1 queue.
For example with this `supervisor` config for creating 2 consumers for `import.module.testModule.queue`.

```

[program:local-import-module-testModule-consumer]
command=php /var/www/site.loc/public/index.php humus amqp consumer import.module.testModule.consumer --without-signals
process_name=%(program_name)s_%(process_num)02d
numprocs=2
directory=/var/www/site.loc/data/
autostart=true
autorestart=true
stdout_logfile=/var/www/site.loc/data/logs/import.module.testModule.exchange.log
stdout_logfile_maxbytes=50MB
stderr_logfile=/var/www/site.loc/data/logs/import.module.testModule.exchange.error.log
stderr_logfile_maxbytes=50MB
```

I can't imagine different usage.